api_docs/actions.json

```diff
@@ -0,0 +1 @@
elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L26"}}],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":19,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L19"},"initialIsOpen":false},{"id":"def-common.ActionTypeExecutorResult","type":"Interface","label":"ActionTypeExecutorResult","signature":[{"pluginId":"actions","scope":"common","docId":"kibActionsPluginApi","section":"def-common.ActionTypeExecutorResult","text":"ActionTypeExecutorResult"},"<Data>"],"description":[],"children":[{"id":"def-common.ActionTypeExecutorResult.actionId","type":"string","label":"actionId","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":34,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L34"}},{"id":"def-common.ActionTypeExecutorResult.status","type":"CompoundType","label":"status","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":35,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L35"},"signature":["\"error\" | \"ok\""]},{"id":"def-common.ActionTypeExecutorResult.message","type":"string","label":"message","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":36,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L36"},"signature":["string | undefined"]},{"id":"def-common.ActionTypeExecutorResult.serviceMessage","type":"string","label":"serviceMessage","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":37,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L37"},"signature":["string | undefined"]},{"id":"def-common.ActionTypeExecutorResult.data","type":"Uncategorized","label":"data","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":38,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L38"},"signature":["Data | undefined"]},{"id":"def-common.ActionTypeExecutorResult.retry","type":"CompoundType","label":"retry","description":[],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":39,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L39"},"signature":["boolean | Date | null | undefined"]}],"source":{"path":"x-pack/plugins/actions/common/types.ts","lineNumber":33,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/types.ts#L33"},"initialIsOpen":false}],"enums":[],"misc":[{"id":"def-common.BASE_ACTION_API_PATH","type":"string","label":"BASE_ACTION_API_PATH","description":[],"source":{"path":"x-pack/plugins/actions/common/index.ts","lineNumber":10,"link":"https://github.com/elastic/kibana/tree/masterx-pack/plugins/actions/common/index.ts#L10"},"signature":["\"/api/actions\""],"initialIsOpen":false}],"objects":[]}}
```

Comment:
Can we enable pretty printing for these too? Would like to be able to make sense of them

Reply:
🤔 hm, what happened there?? Will try this one more time, not sure why it didn't pretty print.